gradle/libs.versions.toml

```diff
@@ -17,6 +17,7 @@ clikt = "3.5.2"
 commonsCompress = "1.23.0"
 cvssCalculator = "1.4.1"
 cyclonedx = "7.3.2"
+diffUtils = "4.12"
 diskLruCache = "2.0.2"
 exposed = "0.41.1"
 flexmark = "0.64.4"
```

Comment:
commit-msg: Please mention that "click to see diff" does not work as of this PR by default anymore.
And that one has to change the value manually "simple".

Also, it'd be nice to have a rationale why the default behavior is changed.

Reply:
> Please mention that "click to see diff" does not work as of this PR by default anymore.

But that's not the case. Kotest's own default is to use [non-simple ("condensed") output for large strings](https://github.com/kotest/kotest/blob/4d11dbf9cc80c3226c6add9eda59f51fa4682db5/kotest-assertions/kotest-assertions-shared/src/commonMain/kotlin/io/kotest/assertions/eq/StringEq.kt#L57), but we did and still do override that with ["simple" when running from IntelliJ](https://github.com/oss-review-toolkit/ort/blob/6c479e2712945b3f9e7a20587a5a741668528c31/build.gradle.kts#L123) (as Kotest's own IntelliJ detection stopped working for newer IntelliJ versions).

In the latest iteration of this PR, the new unified diff is only used of the `kotest.assertions.multi-line-diff` system property is set to "unified" (which is a value that currently only we understand). And that value is only set when running functional tests via Batect.

TL;DR the default behavior did not change for local development with IntelliJ.